### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by 7-Stack**
Please review the project environment setup and initiate the merge if all is good.

### CODE REVIEW

File: src/app.routes/index.routes.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+module.exports = (app) => {
+    app.get(`${basePath}/docs`, (req, res) => {
+        res.redirect()
+    })
+
+    app.get(`${basePath}/health-check`, (req, res) => {
+        res.send({
+            success: true,
+            message: "Health Check"
```
Comment (ezehlivinus) [inline src/app.routes/index.routes.js]: Please change this message to another: maybe use: `Server is up and running`

File: src/middlewares/errorHandler.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                                stackTrace
+                            });
+                            default:
+
+                            return res.status(500).send({
+                                success: false,
+                                message: err.message
+                            });
+    }
+};
```
Comment (ezehlivinus) [inline src/middlewares/errorHandler.js]: Please format your code properly

**Comment by ezehlivinus** [comment]
Also, could you try to name your branch correctly properly next time? Not just feature. @7-Stack

**Review by ezehlivinus — CHANGES_REQUESTED** [review]

File: src/app.routes/index.routes.js (hunk 1 above)
Comment (7-Stack) [reply src/app.routes/index.routes.js]: Ok. I have modified the message to: Server is Live!

File: src/middlewares/errorHandler.js (hunk 2 above)
Comment (7-Stack) [reply src/middlewares/errorHandler.js]: Ok. File has been formatted.

**Review by ezehlivinus — APPROVED** [review]

**Pull request merged**